.github/workflows/ci.yml

```diff
@@ -190,12 +190,8 @@ jobs:
             echo "SM_ENV=dev-tools" >> $GITHUB_ENV
           fi
 
-      - name: Set up Slice Machine project
-        run: |
-          yarn workspaces foreach --include '{@slicemachine/adapter-next,@slicemachine/manager,@slicemachine/plugin-kit,start-slicemachine}' --parallel --topological-dev run build
-          yarn workspace slice-machine-ui pack --out ../../e2e-projects/next/sm.tgz
-          yarn workspace cimsirp add -D ./sm.tgz
-          echo "telemetry=false" > ./e2e-projects/next/.prismicrc
```

Comment:
could you explain why we don't need these anymore?

Reply:
This was made to sync e2e-tests folder with our local code. But the play command is doing that by default, so no need.

BUT I totally forgot that it was also adding telemetry=false, so will add that to the existing setup so e2e tests don't trigger amplitude events.

Reply:
Done: https://github.com/prismicio/slice-machine/pull/1609/commits/842ac739a266e25a9efe1e988d2e5d903073c38f